.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build dist and wheels
+
+on:
+  push:
+    branches:
+      - 'build_wheels'
+
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -U "cython"
+
```

Comment:
Why is cython not in requirements ?

Reply:
it is!

but the build fails if i dont' do that... This is weird.